src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -132,8 +126,6 @@ private function warmupCache(InputInterface $input, OutputInterface $output, $re
      * @param string $warmupDir
      * @param string $realCacheDir
      * @param bool   $enableOptionalWarmers
-     *
-     * @internal to be removed in 4.0
```

Comment:
note that in 3.4, the command class is `@final` so we can still make them private in 4.0. I'll take care of it when merging up.